runtime/include/qio/sys.h

```diff
@@ -65,6 +65,24 @@ typedef struct sys_statfs_s {
   uint64_t    f_namelen;  /* maximum length of filenames */
 } sys_statfs_t;
 
+typedef struct sys_stat_s {
+  dev_t       st_dev;       /* Device ID of device containing file */
+  ino_t       st_ino;       /* File serial number. */
+  mode_t      st_mode;      /* Mode of file (see below). */
+  nlink_t     st_nlink;     /* Number of hard links to the file. */
+  uid_t       st_uid;       /* User ID of file. */
+  gid_t       st_gid;       /* Group ID of file.  */
+  dev_t       st_rdev;      /* Device ID (if file is character or block special). */
+  off_t       st_size;      /* File Size in bytes */
+  off_t       st_atimespec; /* Last data access timestamp.  */
+  off_t       st_mtimespec; /* Last data modification timestamp. */
+  off_t       st_ctimespec; /* Last file status change timestamp. */
```

Comment:
I was expecting st_atim, st_mtim, st_ctim fields of type `struct timespec` per the newer POSIX: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_stat.h.html

If you have these, we can make paretheses-less methods in Chapel that allow access to st_ctime etc. I also don't know why you named them ending in 'timespec'.